src/api/app/views/webui/package/responsive_ux/show_actions/_trigger_services.html.haml

```diff
@@ -1,4 +1,5 @@
 %li.nav-item
-  = link_to(package_trigger_services_path(package: package, project: project), method: :post, class: 'nav-link') do
+  = link_to(package_trigger_services_path(package: package, project: project), method: :post, class: 'nav-link',
+            title: 'Trigger Services') do
     %i.fas.fa-project-diagram.fa-lg.mr-2
-    Trigger services
+    %span.nav-item-name Trigger Services
```

Comment:
I like how you pay attention to details by fixing the capitalization :+1:

Reply:
Thanks. I'm trying to make it consistent:)